### PR TITLE
[MeL] Changes in MeshLib used for GeoMapper refactorization.

### PR DESCRIPTION
**Pull request by TomFischer**
See commit messages for details.

### CODE REVIEW

File: MeshLib/MeshSearch/MeshElementGrid.h [hunk 1]
```diff
@@ -72,6 +72,8 @@ class MeshElementGrid final {
 		return elements_vec;
 	}
 
+	MathLib::Point3d const& getGridMinEdgePoint() const;
```
Comment (norihiro-w) [inline MeshLib/MeshSearch/MeshElementGrid.h]: I'm not sure what "GridMinEdgePoint" means. can you add documentation or rename it?

**Comment by bilke** [comment]
Jenkins test this please.

**Comment by TomFischer** [comment]
@norihiro-w wrote:
> I'm not sure what "GridMinEdgePoint" means. can you add documentation or rename it?

Renamed it and added documentation, see commit f4be806.

**Comment by norihiro-w** [comment]
:+1:

**Comment by endJunction** [comment]
Jenkins, test this please!

**Pull request merged**